README.rst

```diff
@@ -124,6 +124,16 @@ file within that directory contains a refresh token for the user who authorized
 themselves. This refresh token should be treated as if it were a password and
 not shared or otherwise disclosed!
 
+To find the course id, item id, and part id, first go to the web authoring
+interface for your course. There, the URL will of the form:
```

Comment:
"web authoring interface for your programming assignment" maybe?  The url you mention below is item specific.

Also, "the url will *be* of the form"